### PR TITLE
Extend GetOperations api with mtaId query param

### PR DESCRIPTION
**Pull request by radoslav-d**
Extend the GetOperations api to have a new query param for the mta id.
This will allow more native querying for operation by mta id.
JIRA: LMCROSSITXSADEPLOY-1723 and LMCROSSITXSADEPLOY-1578

### CODE REVIEW

File: com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java [hunk 1]
```diff
@@ -91,9 +91,10 @@
         }) }, tags = {})
     @ApiResponses(value = { @ApiResponse(code = 200, message = "OK", response = Operation.class, responseContainer = "List") })
     public ResponseEntity<List<Operation>> getOperations(@PathVariable(PathVariables.SPACE_GUID) String spaceGuid,
+                                                         @RequestParam(name = PathVariables.MTA_ID, required = false) String mtaId,
```
Comment (enchobelezirev) [inline com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java]: Is MTA_ID in this case path variable?

File: com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java [hunk 2]
```diff
@@ -91,9 +91,10 @@
...
                                                          @RequestParam(name = "last", required = false) Integer last,
```
Comment (enchobelezirev) [inline com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java]: Why this "last" and "state" are not presented as contants but the MTA_ID is ?

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImpl.java [hunk 3]
```diff
@@ -84,9 +84,9 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(OperationsApiServiceImpl.class);
 
     @Override
-    public ResponseEntity<List<Operation>> getOperations(String spaceGuid, List<String> stateStrings, Integer last) {
+    public ResponseEntity<List<Operation>> getOperations(String spaceGuid, String mtaId, List<String> stateStrings, Integer last) {
         List<Operation.State> states = getStates(stateStrings);
-        List<Operation> operations = filterByQueryParameters(last, states, spaceGuid);
+        List<Operation> operations = filterByQueryParameters(mtaId, last, states, spaceGuid);
```
Comment (enchobelezirev) [inline com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImpl.java]: Could you put it as last param? For me, it is more intuitevly to add new method parameters in the last positions?

File: com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java (hunk 1 above)
Comment (radoslav-d) [reply com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java]: no, it is query param

File: com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java (hunk 2 above)
Comment (radoslav-d) [reply com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/OperationsApi.java]: will add them

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImpl.java (hunk 3 above)
Comment (radoslav-d) [reply com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/api/impl/OperationsApiServiceImpl.java]: sure

**Pull request merged**